ie_serving/main.py

```diff
@@ -62,20 +63,41 @@ def check_config_structure(configs):
         sys.exit()
 
 
+def get_model_spec(config):
+
+    model_name = config.get('name', config.get('model_name', None))

```

Comment:
This lines are specific only for `one model` path which should be in `parse_one_model` function.